packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -47,12 +47,26 @@ abstract class ConstrainedLayoutBuilder<ConstraintType extends Constraints> exte
   /// The builder must not return null.
   final Widget Function(BuildContext context, ConstraintType constraints) builder;
 
+  /// Whether [builder] needs to be called again even if the layout constraints
+  /// are the same.
+  ///
+  /// When this widget is rebuilt, it is most likely that we need to call
```

Comment:
I find the wording "rebuilt" here a little confusing since this callback isn't on the `Element.rebuild` path. Maybe we can reword this to make it clear that this callback is consulted when the widget configuration is changed (i.e. the `Element.update` path)?

Reply:
Done